packages/web-api/src/methods.ts

```diff
@@ -506,6 +506,7 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     access: bindApiCall<OAuthAccessArguments, OauthAccessResponse>(this, 'oauth.access'),
     v2: {
       access: bindApiCall<OAuthV2AccessArguments, OauthV2AccessResponse>(this, 'oauth.v2.access'),
+      exchange: bindApiCall<OAuthV2AccessArguments, OauthV2AccessResponse>(this, 'oauth.v2.exchange'),
```

Comment:
TODO: Need to have `OAuthV2ExchangeArguments` for this. The arguments are not the same with `oauth.v2.access`. It has `token`. Regarding the response, it's fine to use `WebAPICallResult` for now.